build_with_cmake/node-addon-api/hello.cc

```diff
@@ -0,0 +1,14 @@
+#include <napi.h>
+
+Napi::String Method(const Napi::CallbackInfo& info) {
```

Comment:
This method should be static.

Reply:
Fixed in https://github.com/nodejs/node-addon-examples/pull/114.